freqtrade/freqai/base_models/BaseClassifierModel.py

```diff
@@ -50,21 +50,30 @@ def train(
         logger.info(f"-------------------- Training on data from {start_date} to "
                     f"{end_date} --------------------")
         # split data into train/test data.
-        data_dictionary = dk.make_train_test_datasets(features_filtered, labels_filtered)
+        d = dk.make_train_test_datasets(features_filtered, labels_filtered)
```

Comment:
Really, I'd prefer to shorten the keys of the `data_dictionary`, but I think it would break too many people's code doing so.

But we can use `ds` instead.

Reply:
yeah, user facing stuff (like keys) are very hard to change